alacritty/src/main.rs

```diff
@@ -62,7 +62,7 @@ use crate::event::{Event, Processor};
 #[cfg(target_os = "macos")]
 use crate::macos::locale;
 
-fn main() {
+fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
I've only ever gotten extremely poor results from doing this. I don't think it's a great idea?

Reply:
I think that's fine, like it'll print error? At least that's how I've parsed your lets return `Result`, but now I see that you wanted to return it from `run_return`, but you can't do that.

I'm not sure that this is any different than then `println` and then `exit` that what it should do anyway, no?

Reply:
Hm, I think my main issue with last time I tried it is that you don't get any backtrace by doing this. Which should be fine in our case since we were just using `println` anyway.